spec/controllers/animals_ controller_spec.rb

```diff
@@ -68,6 +74,23 @@
           end
         end
 
+        context 'con la fecha de nacimiento posterior a la fecha de ingreso' do
+        
+          let(:birthdate)  { Faker::Date.between(3.days.ago, Date.today) }
+          let(:admission_date) { Faker::Date.between(1.year.ago, Date.today) }
+
+          it 'devuelve un estado 422' do
+            request.headers['X-USER-TOKEN'] = user.authentication_token
```

Comment:
Para este caso, y en otros veo que se repite `request.headers['X-USER-TOKEN'] = user.authentication_token` dentro de todos los it dentro del context. Podrias ponerlo dentro de un bloque `before` donde corresponda si es posible